imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -65,24 +71,27 @@ class CaseExplorer extends Component {
       })
     }
   }
+
   render () {
     const { isLoading, dispatch, match } = this.props
-    const { unitsDict, showOpen, expandedUnits } = this.state
+    const { unitsDict, showOpen, expandedUnits, assignedToMe } = this.state
     const casesFilterFunc = showOpen ? x => !isClosed(x) : x => isClosed(x)
-
     return (
       <div className='flex flex-column roboto overflow-hidden flex-grow h-100'>
         <div className='bb b--black-10 overflow-auto flex-grow'>
           <div className='flex pl3 pv3 bb b--very-light-gray'>
             <div onClick={() => this.handleStatusClicked(true)} className={'f6 fw5 ' + (showOpen ? 'mid-gray' : 'silver')}> Open </div>
             <div onClick={() => this.handleStatusClicked(false)} className={'f6 fw5 ml4 ' + (!showOpen ? 'mid-gray' : 'silver')}> Closed </div>
+            <div onClick={() => this.handleAssignedClicked()} className={'f6 fw5 ml4 ' + (assignedToMe ? 'mid-gray' : 'silver')}> Assigned To Me</div>
           </div>
           {!isLoading && Object.keys(unitsDict)
             .reduce((all, unitTitle) => {
               const isExpanded = expandedUnits.includes(unitTitle)
               const allCases = unitsDict[unitTitle]
-              const hasCases = allCases.filter(casesFilterFunc)
-              if (hasCases.length > 0) {
+              const myCaseFilter = allCases.filter(caseItem => caseItem.assignee === assignedToMe)
```

Comment:
Does this actually work? in the current version of the code, `assignedToMe` will be the who;e user object, while `assignee` will be the user's BZ login email (a string).

Reply:
`myCaseFilter` is not a very descriptive name for this const by the way. Since it'll hold an array of the cases assigned to the user, it will be better to name it something like `myCases` or `assignedCases`

Reply:
A click handler updates `assignedToMe` with bugzillaCreds.login (email address).  I used BZ login email from `assignedToMe` state to filter assigned cases. Now, `assignedToMe` is a boolean value. I will filter assigned cases with a new variable. `myCases` is renamed as `assignedCases`.  Thank you!